lego/apps/reactions/exceptions.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework import status
+from rest_framework.exceptions import APIException
+
+
+class APIReactionExists(APIException):
+    status_code = status.HTTP_409_CONFLICT
+    default_detail = "Reaction already exists."
+
+
+class APITooManyReactions(APIException):
+    status_code = status.HTTP_413_REQUEST_ENTITY_TOO_LARGE
```

Comment:
This status code is maybe a bit weird? 🤔

Reply:
I went through a few status codes to look for a proper code for it. What would you suggest? It's for when a user has reacted too many times in a content object. I considered using 409 Conflict instead, but I don't believe that's fitting either.

Reply:
I do agree 413 isnt really fitting for a real REST api, so I'm open to suggestions.

Reply:
`409` is reasonable, but I don't really care that much about return codes (as they are mostly stupid). As long as the message is ok, and in this case it is, it doesn't really matter.

As long as forms utilize the form validation data in the cases it makes sense, everything should be ok.

Reply:
I'm keeping it as `413` for now, as I use it for error handling for the frontend.